pkg/vz/network_darwin.go

```diff
@@ -27,7 +28,7 @@ func PassFDToUnix(unixSock string) (*os.File, error) {
 	if err != nil {
 		return nil, err
 	}
-	err = fd.Put(unixConn.(*net.UnixConn), server)
```

Comment:
I don't think we have to introduce a new `netutil` package just for avoiding this single type assertion.